modules/monitoring-modifying-retention-time-and-size-for-prometheus-metrics-data.adoc

```diff
@@ -59,6 +71,7 @@ data:
   config.yaml: |
     prometheusK8s:
       retention: *24h*
+      retentionSize: *10GB*
 ----
 
 ** *To modify the retention time for the Prometheus instance that monitors user-defined projects*:
```

Comment:
the example also set retention size, change to
`To modify the retention time and size for the Prometheus instance that monitors user-defined projects`

Reply:
fixed